tests/php-code-style/.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude('oql')
+	->in(__DIR__.'/addons')
+	->in(__DIR__.'/application')
+	->in(__DIR__.'/core')
+	->in(__DIR__.'/datamodels')
+	->in(__DIR__.'/dictionaries')
+	->in(__DIR__.'/pages')
+	->in(__DIR__.'/portal')
+	->in(__DIR__.'/setup')
+	->in(__DIR__.'/sources')
+	->in(__DIR__.'/synchro')
+	->in(__DIR__.'/tests')
+    ->in(__DIR__ . '/webservices')
```

Comment:
```suggestion
$finder = PhpCsFixer\Finder::create()
	->in(__DIR__)
    ->exclude(__DIR__.'/core/oql');
```

Is it possible to do something based on opt-out rather than opt-in? It would be easier to maintain and to read.
Also, doesn't this exclude clause risk to exclude also all "oql" folders rather than the one `/core/oql`?

Reply:
just fixed this feedback as well